### PR TITLE
Fixes #4028 - add *-computer cmdlets to breaking changes

### LINKED ISSUE #4028: Add-Computer is not included in PS v6
That information is missing from the Breaking Changes docs.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 6dd43f90-52f6-78d3-c2c1-b293c06555d4
* Version Independent ID: 654786a4-ff34-d57a-eca4-30c5ba2b5a7f
* Content: [Breaking Changes for PowerShell 6.0](https://docs.microsoft.com/en-us/powershell/scripting/whats-new/breaking-changes-ps6?view=powershell-6#feedback)
* Content Source: [reference/docs-conceptual/whats-new/breaking-changes-ps6.md](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/docs-conceptual/whats-new/breaking-changes-ps6.md)
* Product: **powershell**
* GitHub Login: @joeyaiello
* Microsoft Alias: **jaiello**

### PR DESCRIPTION
**Pull request by sdwheeler**
Version(s) of document impacted
------------------------------
- [ ] Impacts 6.next document
- [x] Impacts 6 document
- [ ] Impacts 5.1 document
- [ ] Impacts 5.0 document
- [ ] Impacts 4.0 document
- [ ] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [x] The documented feature was introduced in version (list version here) of PowerShell
- [ ] This issue only shows up in version (list version(s) here) of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

### CODE REVIEW

File: reference/docs-conceptual/whats-new/breaking-changes-ps6.md [hunk 1]
```diff
@@ -64,6 +64,16 @@ functionality with new functionality and a redesigned syntax:
 Due to the use of unsupported APIs, `Microsoft.PowerShell.LocalAccounts` has been removed from
 PowerShell Core until a better solution is found.
 
+### `*-Computer` cmdlets
+
+Due to the use of unsupported APIs, the following cmdlets been removed from PowerShell Core until a
```
Comment (DCtheGeek) [inline reference/docs-conceptual/whats-new/breaking-changes-ps6.md]: "cmdlets **have** been removed"

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[940501c](https://github.com/sdwheeler/PowerShell-Docs/pull/4031/commits/940501c5cad401fb5e29cb69e47df777cbfc1ab9)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/docs-conceptual/whats-new/breaking-changes-ps6.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4028/reference/docs-conceptual/whats-new/breaking-changes-ps6.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/breaking-changes-ps6?view=powershell-3.0&branch=pr-en-us-4031) (powershell-3.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/breaking-changes-ps6?view=powershell-4.0&branch=pr-en-us-4031) (powershell-4.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/breaking-changes-ps6?view=powershell-5.0&branch=pr-en-us-4031) (powershell-5.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/breaking-changes-ps6?view=powershell-5.1&branch=pr-en-us-4031) (powershell-5.1)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/breaking-changes-ps6?view=powershell-6&branch=pr-en-us-4031) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C22%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201903221748456786-4031%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=JKnwZT7Bvj2cj15u03wkdxZjobzVU0dp0reaogbuKiQ%3D&st=2019-03-22T17%3A48%3A43Z&se=2019-04-22T17%3A53%3A43Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/docs-conceptual/whats-new/breaking-changes-ps6.md (hunk 1 above)
Comment (sdwheeler) [reply reference/docs-conceptual/whats-new/breaking-changes-ps6.md]: Fixed. And, yes, this is targeted at staging. These were removed in the PS6.0 release.

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[e28d274](https://github.com/sdwheeler/PowerShell-Docs/pull/4031/commits/e28d274df69e0775d14e42c20e34d2a08703a913)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/docs-conceptual/whats-new/breaking-changes-ps6.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4028/reference/docs-conceptual/whats-new/breaking-changes-ps6.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/breaking-changes-ps6?view=powershell-3.0&branch=pr-en-us-4031) (powershell-3.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/breaking-changes-ps6?view=powershell-4.0&branch=pr-en-us-4031) (powershell-4.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/breaking-changes-ps6?view=powershell-5.0&branch=pr-en-us-4031) (powershell-5.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/breaking-changes-ps6?view=powershell-5.1&branch=pr-en-us-4031) (powershell-5.1)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/whats-new/breaking-changes-ps6?view=powershell-6&branch=pr-en-us-4031) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C22%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201903221814131576-4031%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=kUj8mJjn%2F6GA0wwY17U77mVfCsqfzYR4yacGoO7Fv84%3D&st=2019-03-22T18%3A13%3A58Z&se=2019-04-22T18%3A18%3A58Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**